server/sonar-server/src/test/java/org/sonar/server/issue/index/IssueIndexTest.java

```diff
@@ -586,6 +591,7 @@ public void filter_by_created_after_and_before() {
       .build(), new SearchOptions());
   }
 
+  @Ignore
   @Test
   public void filter_by_create_after_and_before_take_into_account_timezone() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)